### ISSUE TITLE
Betaflight f3 board stuck on "detected device with total flash size 256kib"  

### ISSUE THREAD

**Issue by Ub6ib9** [OpenRA/OpenRA#13981]
Can't flash betaflightf3 board even in dfu mode. I have tried everything. I don't want to upgrade but every QWAD I plug in to betaflight starts having issues with not recognizing the receivers. And only got one of my quads flashed. And 2 are paperweights now. And I lost one I don't normally fly because I had to resort to it. Now I'm pissed. All because the receiver tabs are having issues with all firmware except for the most recent version. I shouldn't be stuck having to flash newest versions on all QWAD's. 

Why all of a sudden nothing is working correctly!

**Comment by reaperrr** [comment]
Nothing OpenRA-related.

**Comment by Ub6ib9** [comment]
Exactly same thing happened on 3 drones that were previously set up and I was just going to tweak settings and then I went into the receiver tab and it was only showing movement (and not from the TX) on the roll pitch and yaw slider bars flashing on and off from 0 to 1500 all other channels had nothing. I found a thread that said that I had to update to the most recent stable version and so I. Tried on all 3 only one flashed (sprf3) and the other two (bff3 would go into dfu mode but hung on the detected device with total flash size 256kib) and the flip 32 f4 omnibusf4 target gets into dfu but not recognizing as so on the pc unlike the bff3. But both connect fine... I don't see how I can have issues with some but fine on others. And they were all previously setup and flying fine.
